docs/cli.rst

```diff
@@ -116,7 +116,7 @@ Last but not least, you can still use all PhantomJS standard CLI options as you
 
 .. hint::
 
-   To remember what the native phantomjs available cli options are, run the ``phantomjs --help`` command.
+   To remember what the native PhantomJS cli options are available, run the ``phantomjs --help`` command.
```

Comment:
To ~~remember~~ display what ~~the~~ native PhantomJS cli options are available, ...